.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
 - sudo touch /etc/apt/sources.list.d/dotnetdev.list
 - echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" | sudo tee -a /etc/apt/sources.list.d/dotnetdev.list
 - sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893 && sudo apt-get update
-- sudo apt-get install -y --no-install-recommends dotnet-dev-1.0.0-preview2.1-003177
+- sudo apt-get install -y --no-install-recommends dotnet-dev-2.1.0-preview1-006330
```

Comment:
Do we need the preview? Why not `sudo apt-get install -y --no-install-recommends dotnet-sdk-2.0.0` and reverting `global.json`

Reply:
Thanks. Just found proper sdk version.